app/Http/Requests/Admin/Fields/CreateFieldRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Requests\Admin\Fields;
+
+use App\Models\Field;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateFieldRequest extends FormRequest {
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize() {
+        return true;
```

Comment:
we should add a policy that uses admins https://laravel.com/docs/5.7/authorization#generating-policies - since we already created an admin model

Reply:
ill do that next?